client/my-sites/email/email-providers-comparison/email-provider-card.jsx

```diff
@@ -114,7 +114,7 @@ EmailProviderCard.propTypes = {
 	badge: PropTypes.object,
 	description: PropTypes.string,
 	formattedPrice: PropTypes.node,
-	discount: PropTypes.string,
```

Comment:
Not blocking, but it might have been easier to handle this in a separate PR.

Reply:
I'll revert :) I'll create Asana card too, as there is an ugly error in console.